lib/pinch-zoom.ts

```diff
@@ -112,6 +118,32 @@ export default class PinchZoom extends HTMLElement {
     this.addEventListener('wheel', event => this._onWheel(event));
   }
 
+  attributeChangedCallback(name: string, oldValue: string, newValue: string) {
+    const parseMinScale = (attrValue: string) => {
+      if (!attrValue) return MIN_SCALE;
+
+      const value = parseFloat(attrValue);
+      if (Number.isFinite(value)) return Math.max(MIN_SCALE, value);
+
+      return MIN_SCALE;
+    }
```

Comment:
This function is created every time one of the observed attributes changes. Are you sure this isn't more expensive than running these steps in the getter.

I don't think `minScale` is going to be get'd enough to make it worth this complexity.

Reply:
I can move it out.

The problem is that you check minScale each time you apply a scale, so it would be nice to avoid the conversion at each of these steps.

Reply:
Let's remove it unless it can be shown to be a bottleneck.

Reply:
OK, I move it back into the getter then and drop the caching

Reply:
Done